sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -6,11 +6,12 @@
 - New `<evaluator>.binary_aggregate` field added to evaluation result metrics. This field contains the aggregated binary evaluation results for each evaluator, providing a summary of the evaluation outcomes.
 
 ### Breaking Changes
+- In the experimental RedTeam's scan method, the `data_only` param has been replaced with `_skip_evals` and if you do not want data to be uploaded, use the `skip_upload` flag.
```

Comment:
Python convention is to use "_" for private params, classes etc, if we want to make it public should we name it "skip_evals" ?